src/omni_epd/displays/inky_display.py

```diff
@@ -108,19 +120,23 @@ def __init__(self, deviceName, config):
         self.width = self._device.width
         self.height = self._device.height
 
+        # get colors from the inky lib (won't be used normally as inky does conversion)
+        self.palette_filter = deviceObj.DESATURATED_PALETTE

```

Comment:
I realize now that's be the right way to do that. Can be applied to `clear()` as well.

Reply:
Good pt, I just sort of realized it as I was updating it that it would work that way.